internal/manifests/manifests_test.go

```diff
@@ -791,6 +792,30 @@ spec:
 			Expect(responsePayload.Payload.Folder).To(Equal(defaultFolder))
 		})
 	})
+
+	Context("Manifest path parsing functions", func() {
```

Comment:
Use `Describe`  with the function you're testing and `It` for individual behaviors you're testing. So something like

```go
Describe("ParsePath", func() {
  It("parses a path into folder and filename", func() {
    ...
  })
  It("fails when provided a non-manifest path", func() {
    ..
  })
})
Describe("IsManifest", func() {
  ...
})
```

This way a test failure will show exactly which function is failing and what about it isn't working.